### PR TITLE
[FR] Enable other clouds

### PR DESCRIPTION
**Pull request by maririos**
Notes:
- AAD tests will fail because of issue https://github.com/Azure/azure-sdk-for-net/issues/17192 . Still worth to enable the API Key test.
- Service team asked us to test in `canadacentral` so doing "custom" CloudConfig

### CODE REVIEW

File: sdk/formrecognizer/tests.yml [hunk 1]
```diff
@@ -5,3 +5,20 @@ extends:
   parameters:
     ServiceDirectory: formrecognizer
     TimeoutInMinutes: 90
+    CloudConfig:
```
Comment (benbp) [inline sdk/formrecognizer/tests.yml:8]: You're missing preview in SupportedClouds, so you can either add it there or remove it from this config.

Comment (benbp) [reply sdk/formrecognizer/tests.yml:8]: Unless the idea is to be able to enable it for testing.

Comment (maririos) [reply sdk/formrecognizer/tests.yml:8]: Thank you. Service only manages prod and canary. what is the idea behind preview?
For now I am removing it from the configuration

Comment (benbp) [reply sdk/formrecognizer/tests.yml:8]: It's another test subscription we maintain that enables some feature flags for preview features in ARM. Some services need it for certain tests.

Comment (maririos) [reply sdk/formrecognizer/tests.yml:8]: Ahh I see, yeah we don't need it then

**Comment by maririos** [comment]
@benbp is there something else that I should change or are we good to go here

**Review by benbp — APPROVED** [review]

**Pull request merged**